packages/react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -129,11 +129,14 @@ export class DetailsColumnBase extends React.Component<IDetailsColumnProps> {
                       <IconComponent className={classNames.iconClassName} iconName={column.iconName} />
                     )}
 
-                    {column.isIconOnly ? (
-                      <span className={classNames.accessibleLabel}>{column.name}</span>
-                    ) : (
-                      column.name
-                    )}
+                    {column.onRenderHeader && column.onRenderHeader()}
```

Comment:
```ts
function defaultOnRenderHeader(props: IDetailsColumnProps): JSX.Element | null {
  const { column } = props;

  return column.isIconOnly ? (
    ___
  ) : column.name;
}
...
const onRenderHeader = column.onRenderHeader ? composeRenderFunction(column.onRenderHeader, defaultOnRenderHeader) : defaultOnRenderHeader;
```

Reply:
Thanks for the code. This should be fixed now.